pulsar-metadata/src/main/java/org/apache/pulsar/metadata/cache/impl/MetadataCacheImpl.java

```diff
@@ -243,6 +243,17 @@ public CompletableFuture<Void> create(String path, T value) {
         return future;
     }
 
+    @Override
+    public CompletableFuture<Void> put(String path, T value) {
+        final byte[] bytes;
+        try {
+            bytes = serde.serialize(path, value);
+        } catch (IOException e) {
+            return CompletableFuture.failedFuture(e);
+        }
+        return store.put(path, bytes, Optional.empty()).thenAccept(__ -> {});
```

Comment:
It seems that the possible already cached value should be expired or updated too?

Reply:
`.thenAccept(__ -> refresh(path))` would be aligned how the other operations handle cache invalidation and refreshing.